src/Acuminator/Acuminator.Vsix/Formatter/FormatBqlCommand.cs

```diff
@@ -24,7 +26,7 @@ internal sealed class FormatBqlCommand
 		/// <summary>
 		/// Command ID.
 		/// </summary>
-		public const int CommandId = 0x0100;
+		public const int CommandId = 0x0101;
```

Comment:
Why is that?

Reply:
I took it from the example with submenu where submenu already had that ID. So my submenu has ID 0x0100  and its commands has IDs starting from 0x0101.